src/@types/parseable/api/stream.ts

```diff
@@ -7,6 +7,11 @@ export type LogStreamSchemaData = {
 	metadata: Record<string, string>;
 };
 
+export type LogStreamQueryWithFields = {
+	fields: string[];
+	records: Record<string, string>[];
```

Comment:
There is an existing `Log` type defined. Use that here.

Reply:
What is the type?

Reply:
so the type here is a array of objects, there is a pre existing type that aligns with our response.